### PR TITLE
feat(*): add wrapper array class samples(waiting for dubbo-go-hessian upgrade)

### PR DESCRIPTION
**Pull request by Mulavar**

### CODE REVIEW

**Review by AlexStocks — APPROVED** [review]

**Comment by LaurenceLiZhixin** [comment]
I guess this pr is waiting for hessian upgreade? @Mulavar

**Comment by Mulavar** [comment]
> I guess this pr is waiting for hessian to upgrade? @Mulavar

you are right.

**Review by cjphaha — APPROVED** [review]

File: rpc/dubbo/go-server/pkg/wrapper_array_class_provider.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+package pkg
+
+import (
+	"context"
+)
+
+import (
+	"dubbo.apache.org/dubbo-go/v3/common/logger"
+
+	hessian "github.com/apache/dubbo-go-hessian2"
```
Comment (zouyx) [inline rpc/dubbo/go-server/pkg/wrapper_array_class_provider.go:27]: format？

**Comment by LaurenceLiZhixin** [comment]
@Mulavar 暂时不能合并的话，前面加个[WIP]

File: rpc/dubbo/go-server/pkg/wrapper_array_class_provider.go (hunk 1 above)
Comment (Mulavar) [reply rpc/dubbo/go-server/pkg/wrapper_array_class_provider.go:27]: 已经 format 过了，都是第三方依赖，且不是同一个，中间用空行分隔。

**Comment by zouyx** [comment]
I have turn it to draft

**Review by cjphaha — APPROVED** [review]

**Review by zhaoyunxing92 — APPROVED** [review]

**Pull request merged**